pkg/operator/ceph/csi/spec.go

```diff
@@ -52,7 +52,8 @@ type Param struct {
 	ResizerImage                             string
 	DriverNamePrefix                         string
 	KubeletDirPath                           string
-	CsiLogDirPath                            string
+	CsiRootPath                              string
+	CsiRootProviderPath                      string
```

Comment:
we dont need to add CsiRoothPath to the param , we can just have `CSILogRootPath` instead of `CsiRootProviderPath`

Reply:
@Madhu-1 
1) I am getting DataDirHostPath value from the cephcluster 
`	CSIParam.CsiRootPath = spec.DataDirHostPath`

2) Updating a CsiRootPath Param value

3) And then using the `CsiRootPath Param value` to append to `CsiRootProviderPath` 

```
tp.CsiRootProviderPath = tp.CsiRootPath + "/" + NFSDriverName
```

If there is a better way to do it please suggest, I am looking for that.

Reply:
Maybe a global varibale, but better with struct.

Reply:
Or maybe I should pass the `CsiRootPath` to  `	err = r.validateAndConfigureDrivers(serverVersion, ownerInfo)`, but which will require a whole lot of functional changes

Reply:
https://github.com/rook/rook/blob/c29d0f9a104c2470efc9535fb6b02b2b0e511939/pkg/operator/ceph/csi/spec.go#L46-L107 this for the configuration which needs to be replaced in the templates, if you are not planning to have `CsiRootPath` in yaml files we don't need to put it here, we can have a variable for it as we do for other configurations.

Reply:
Done